prow/overlays/smaug/service-monitor.yaml

```diff
@@ -6,10 +6,12 @@ metadata:
     monitor-component: operate-first
 spec:
   endpoints:
-    - interval: 30s
+    - path: /metrics
       port: metrics
-      scheme: https
+      scheme: http
```

Comment:
Should scheme be included here, and should it be `http`? Our other service monitors [see search for examples](https://github.com/operate-first/apps/search?q=serviceMonitor) don't specify a scheme at all.
```suggestion
```

Reply:
Maybe it defaults to `http`, but before it was `https` and would not work with the exposed services. So I would rather keep it explicit